### DISCUSSION TITLE
Host the git repo on sites other than GitHub if it's not already.

### DISCUSSION THREAD

**Discussion by MoleMad** [0xERR0R/blocky#922] [Ideas]
Not to sound too conspiratorial, but it might not be the best idea to have a piece of software that's anti-corporate by nature, and only have it hosted on a site owned by Microsoft.

**Comment by kwitsch** [comment]
Sounds reasonable.

If GitHub stays main and the other origin is only kept as backup it would be easy keeping them in sync with some [actions](https://github.com/wangchucheng/git-repo-sync).

**Comment by 0xERR0R** [comment]
Agreed with @kwitsch, GitHub must stay the main repository with PRs, build actions etc. We can just mirror the git repo. I would suggest to take some non-commercial provides, like codeberg (https://codeberg.org/) and not GitLab/BitBucket etc.

Docker images are already mirrored (main is docker.io, mirror is ghcr.io).

**Reply by MoleMad** [reply]
sourcehut.org is another good one which I believe is non commercial.

**Comment by 0xERR0R** [comment]
I created a mirror on codeberg, since I already had a user: https://codeberg.org/0xERR0R/blocky
